cookbook/docs/conf.py

```diff
@@ -270,6 +271,7 @@ def __call__(self, filename):
 gallery_dirs = [
     "auto/core/flyte_basics",
     "auto/core/control_flow",
+    "auto/core/scheduled_worflows",
```

Comment:
how do you reach here, i think we will have to add it to the panels page? and use the `clock` icon

Reply:
Done. 
https://flyte--413.org.readthedocs.build/projects/cookbook/en/413/